Sources/TuistAcceptanceTesting/TuistAcceptanceTestCase.swift

```diff
@@ -12,6 +13,9 @@ public enum Destination {
     case simulator, device
 }
 
+// swiftlint:disable:next force_try
+public let cacheDirectory = try! TemporaryDirectory(removeTreeOnDeinit: true)
```

Comment:
What about scoping temporary directories to individual test cases?

Reply:
I ended up using the temporary directory created with the `MockEnvironment` 👍